deploy/kube/README.md

```diff
@@ -0,0 +1,77 @@
+# Istio Mixer K8s Configuration
+
+These configurations stand up an instance of the mixer, a Prometheus instance that scapes metrics the mixer exposes, and
+a Grafana instance to render those metrics. These are intended for a demo, not a real production setup. The easiest way
+to stand up these deployments is to run (from the Istio mixer root directory):
+
+    $ kubectl create configmap testdata-config --from-file=testdata/globalconfig.yml --from-file=testdata/serviceconfig.yml --from-file=testdata/prometheus.yaml
+    $ kubectl apply -f ./deploy/kube/
+    
+If you're using something like [minikube](https://github.com/kubernetes/minikube) to test locally, you can call the
+mixer server with our test client (`mixc`), providing some attribute values:
+
+    $ MIXS=$(minikube service mixer --url --format "{{.IP}}:{{.Port}}")
+    $ bazel run //cmd/client:mixc -- check -m $MIXS -a source.name=source,target.name=target,api.name=myapi,response.code=200,response.latency=100,client.id=$USER
+
+## mixer.yaml
+`mixer.yaml` describes a deployment and service for the mixer server binary. The deployment annotates the pods it
+creates with a `prometheus_port` annotation which is required by our Prometheus deployment (see [prometheus.yaml](#prometheus)
+below as well as the prometheus configuration in `//testdata/prometheus.yaml`).
+
+Two pieces of configuration are required to run the mixer: a global configuration and a service configuration. Example
+configurations can be found in the `//testdata` directory (`//` indicates the root of the Istio Mixer project directory).
+These configurations are expected to be mounted in two files at `/etc/opt/mixer/`. We use a configmap named `testdata-config`
+to provide these configurations. Usually this configmap is created from the `//testdata/` directory by:
+
+<a name="configmap_command"></a>
+
+    $ kubectl create configmap testdata-config --from-file=testdata/globalconfig.yml --from-file=testdata/serviceconfig.yml --from-file=testdata/prometheus.yaml
+
+A file can be created to capture this configmap by running:
+
+    $ kubectl get configmap testdata-config -o yaml > ./testdata-config.yaml
+
+We do not check in this file since we consider the yaml files in `//testdata` the source of truth for this config.
+
+## grafana.yaml
+`grafana.yaml` describes a deployment and service for Grafana, based on the [`grafana/grafana` docker image](https://hub.docker.com/r/grafana/grafana/).
+The Grafana UI is exposed on port `3000`. This configuration stores Grafana's data in the `/data/grafana` dir, which we
+configure to be an ephemeral directory. To persist your Grafana configuration for longer than the lifetime of the `grafana`
```

Comment:
you can also export the config for grafana and reimport it. that might be worth calling out here (and we should probably link to a demo config).